processor/schematransformerprocessor/README.md

```diff
@@ -0,0 +1,25 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Transformer Processor_ is used to convert existing telemetry data or signals to the most recent targeted version of the semantic convention defined as part of the configuration. The latest schema translation file will be bundled as part of the collector build to avoiding having to fetch it at runtime. 
```

Comment:
>to the most recent targeted version

I think we need to allow the user to choose to which version to transform to, not just to the most recent.

Reply:
Translation can go forward and backward and the user should definitely be able to specify the target version.  Indeed, I expect a common use case will be to ensure that telemetry coming in with a new schema version is adapted to the currently expected, but older, schema version.

Reply:
Yes, I did forget to go back and update this part after reading through the spec in more detail. Let me fix this up in a follow up commit :)